src/app/pages/DashboardPage/TransactionsChartCard.tsx

```diff
@@ -29,11 +29,24 @@ const TransactionsChartCardCmp: FC<TransactionsChartCardProps> = ({ chartDuratio
 
   const totalTransactions = data?.data.buckets.reduce((acc, curr) => acc + curr.tx_volume, 0) ?? 0
 
+  const getPercentageDiff = (): number | null => {
+    if (!lineChartData) {
+      return null
+    }
+
+    const [firstBucket, ...restBuckets] = lineChartData
+    const averageRestBuckets =
+      restBuckets.reduce((acc, curr) => acc + curr.volume_per_second, 0) / restBuckets.length
+
+    return ((averageRestBuckets - firstBucket.volume_per_second) / averageRestBuckets) * -100
+  }
+
+  const percentage = getPercentageDiff()
```

Comment:
> Calculate percentage compared to the average history of the graph, which seems like the most logical solution

Uhhh, I did not expect this to show deviation. Don't we want relative change?

Reply:
It is relative change compared to average of the all previous data points? Or am I misinterpreting something.

Reply:
Update: discussed on frontend sync that we actually want relative change (final value - starting value)

Reply:
Updated, so that it reflect the relative change from final to starting value

Reply:
This math is completely wrong

Reply:
My bad, should be good now.